src/database/utilities/createRecord.js

```diff
@@ -394,6 +395,36 @@ const createTransactionItem = (database, transaction, item) => {
   return transactionItem;
 };
 
+/**
+ * Create a Message record. This will be sent to the server and requests tables
+ * when an app is upgraded from some version to another.
+ *
+ * @param {Realm}  database    App-wide database interface
+ * @param {String} fromVersion Which version the app is being upgraded from.
+ * @param {String} toVersion   Which version the app is being upgraded too.
+ */
+
+const createUpgradeMessage = (database, fromVersion, toVersion) => {
+  const syncSiteId = database.getSetting(SETTINGS_KEYS.SYNC_SITE_ID);
+
+  const body = {
```

Comment:
Nit: an alternative way of doing this would be to have body as a parameterised setter on the message object itself. Something like `message.setBody(messageType, messageParams)`. One benefit I can think of of this way is that you could add some validation logic to prevent arbitrary messages that desktop won't understand.